PancakeSwap-Bots/Prediction-Admin-Operatons/src/agent.ts

```diff
@@ -0,0 +1,12 @@
+import abi from "./abi";
+import findings from "./findings";
+import { TransactionEvent } from "forta-agent";
+
+const CAKE_ADDRESS = "0x18B2A687610328590Bc8F2e5fEdDe3b582A49cdA";
```

Comment:
```suggestion
const PANCAKE_PREDICTION_V2_ADDRESS = "0x18B2A687610328590Bc8F2e5fEdDe3b582A49cdA";
```

Suggesting a better name, as someone could believe that this refers to the Cake token address.